specification/vmware/Microsoft.AVS.Management/virtualMachines.tsp

```diff
@@ -18,21 +17,19 @@ interface VirtualMachines {
 
   get is ArmResourceRead<VirtualMachine>;
 
-  // restrictMovement should be a ArmResourceActionNoContentAsync, but does not respond with a 204.
-  #suppress "@azure-tools/typespec-azure-core/no-openapi" "DO NOT COPY - TODO migrate to LRO apis"
   @doc("Enable or disable DRS-driven VM movement restriction")
-  @armResourceAction(Location)
-  @post
-  @extension("x-ms-long-running-operation", true)
-  restrictMovement(
-    ...ResourceInstanceParameters<VirtualMachine>,
-
-    @bodyRoot
-    @doc("The body type of the operation request.")
-    restrictMovement: VirtualMachineRestrictMovement,
-  ): ArmAcceptedLroResponse | ErrorResponse;
+  restrictMovement is ArmResourceActionAsyncBase<
+    VirtualMachine,
+    VirtualMachineRestrictMovement,
+    ArmAcceptedLroResponse,
```

Comment:
What should the lro return here?  This will have the lro returning 'void'

Reply:
`void` is correct.